ingest-upsert-archive-folders/src/main/scala/uk/gov/nationalarchives/ingestupsertarchivefolders/LambdaRunner.scala

```diff
@@ -0,0 +1,41 @@
+package uk.gov.nationalarchives.ingestupsertarchivefolders
+
+import cats.effect.IO
+import uk.gov.nationalarchives.{DADynamoDBClient, DAEventBridgeClient}
+import uk.gov.nationalarchives.dp.client.fs2.Fs2Client
+import uk.gov.nationalarchives.ingestupsertarchivefolders.Lambda.{Config, Dependencies, StepFnInput}
+
+import cats.effect.unsafe.implicits.global
+
+object LambdaRunner extends App {
+  """
```

Comment:
1. This file belongs to the upsert Lambda; did you edit it deliberately?
2. This string isn't saved to a variable